<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherDialog</class>
 <widget class="QDialog" name="WeatherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NeXtRAD Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="weather_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Weather Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="douglas_sea_state_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Douglas Sea State (1 - 8):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="douglas_sea_state_Edit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="wind_speed_Edit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="wind_speed_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wind Speed (knots):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="wind_dir_Edit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="wind_dir_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wind Direction (deg):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="wave_height_Edit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="wave_height_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wave Height (m):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="wave_dir_Edit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="wave_dir_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wave Direction (deg):</string>
   </property>
  </widget>
  <widget class="QLabel" name="wave_period_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wave Period (seconds):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="wave_period_Edit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="air_temp_Edit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="air_temp_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Air Temperature (deg Celsius):</string>
   </property>
  </widget>
  <widget class="QLabel" name="air_pressure_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Air Pressure (mbar):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="air_pressure_Edit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WeatherDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WeatherDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
